### ISSUE TITLE
I am not able to get the screenshot in localhost:7878....

### ISSUE THREAD

**Issue by priyanka748** [healenium/healenium-web#171]
### Describe the problem

Not able to get the screenshot in maven report.. on command report (The url which is getting generated)

### Healenium Web version

healenium-web 3.2.1

### Healenium Backend version

 healenium-back version 3.2.0

### Platform

Java

### Logs appeared during using Healenium

```shell
Not able to get the screenshot in maven report.. on command report (The url which is getting generated)
```

### Additional context

Not able to get the screenshot in maven report.. on command report (The url which is getting generated)

**Comment by ElenaStepuro** [comment]
@priyanka748 I need to clarify: generated report is empty or you only doesn't see the screenshot in it?

**Comment by priyanka748** [comment]
it is empty actually

**Comment by priyanka748** [comment]
![Uploading image.png…]()

**Comment by priyanka748** [comment]
<img width="955" alt="Capture" src="https://user-images.githubusercontent.com/49894816/151998517-cb1ad30c-8984-4cc9-9f69-bebb85790ec3.PNG">

**Comment by ElenaStepuro** [comment]
Interesting...
We'll try to reproduce it and then notify you)

**Comment by priyanka748** [comment]
Any updates @ElenaStepuro ?

**Comment by priyanka748** [comment]
@ElenaStepuro any updates for the report.. can i get screenshots ??

**Comment by ElenaStepuro** [comment]
Hello @priyanka748 Unfortunately, we cannot reproduce this issue.
Can we have a call to see what steps do you do?

**Comment by ElenaStepuro** [comment]
@priyanka748 for me the most convenient day is Tuesday. Could you please send convenient time slots for you on e-mail elena_stepuro@epam.com?

**Comment by priyanka748** [comment]
Okay.. @ElenaStepuro

**Comment by priyanka748** [comment]
Thanks

**Comment by priyanka748** [comment]
Hi Elena.. 
After adding the dependency I have run through cmd.. mvn clean test
still  I am getting an empty report.. can u please check..
<img width="817" alt="Capture" src="https://user-images.githubusercontent.com/49894816/154476411-fbe67086-dab4-4a39-be9a-c8402800e82b.PNG">

**Comment by rgupta95** [comment]
@ElenaStepuro can you please let me know the solution as i am also getting the same issue what current user have. 
I am using latest maven dependency of healenium 3.3.0.

**Comment by BalurQA** [comment]
@ElenaStepuro  I am also facing same issue could you please tell what is the solution for this. Thank you.

![image](https://user-images.githubusercontent.com/32229011/214486417-d0b3945c-68ef-4898-82aa-9e563652505d.png)

Here is my pom.xml

`
<?xml version="1.0" encoding="UTF-8"?>
<project xmlns="http://maven.apache.org/POM/4.0.0"
         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
    <modelVersion>4.0.0</modelVersion>

    <groupId>com.healenium.demo</groupId>
    <artifactId>healenium-selenium-maven-example</artifactId>
    <version>1.0-SNAPSHOT</version>

    <properties>
        <maven.surefire.plugin.version>3.0.0-M5</maven.surefire.plugin.version>
        <allure.maven.version>2.11.0</allure.maven.version>
        <aspectj.version>1.9.4</aspectj.version>
        <allure.version>2.15.0</allure.version>
        <allure.cmd.download.url>
            https://repo.maven.apache.org/maven2/io/qameta/allure/allure-commandline
        </allure.cmd.download.url>
        <maven.compiler.plugin.version>3.8.0</maven.compiler.plugin.version>
        <maven.compiler.source>8</maven.compiler.source>
        <maven.compiler.target>8</maven.compiler.target>
        <hlm.report.mvn.version>1.1</hlm.report.mvn.version>
        <selenium.java.version>4.8.0</selenium.java.version>
        <healenium.version>3.3.0</healenium.version>
        <testng.version>6.14.3</testng.version>
        <webdrivermanager.version>4.4.3</webdrivermanager.version>
    </properties>








    <build>
        <pluginManagement>
            <plugins>
                <plugin>
                    <groupId>org.apache.maven.plugins</groupId>
                    <artifactId>maven-surefire-plugin</artifactId>
                    <version>${maven.surefire.plugin.version}</version>
                    <configuration>
                        <systemPropertyVariables>
                            <allure.results.directory>${project.build.directory}/allure-results
                            </allure.results.directory>
                        </systemPropertyVariables>
                        <argLine>
                            -javaagent:"${settings.localRepository}/org/aspectj/aspectjweaver/${aspectj.version}/aspectjweaver-${aspectj.version}.jar"
                        </argLine>
                    </configuration>
                    <dependencies>
                        <dependency>
                            <groupId>org.aspectj</groupId>
                            <artifactId>aspectjweaver</artifactId>
                            <version>${aspectj.version}</version>
                        </dependency>
                    </dependencies>
                </plugin>
                <plugin>
                    <groupId>io.qameta.allure</groupId>
                    <artifactId>allure-maven</artifactId>
                    <version>${allure.maven.version}</version>
                    <configuration>
                        <reportVersion>${allure.version}</reportVersion>
                        <allureDownloadUrl>
                            ${allure.cmd.download.url}/${allure.version}/allure-commandline-${allure.version}.zip
                        </allureDownloadUrl>
                    </configuration>
                </plugin>

            </plugins>
        </pluginManagement>
        <plugins>
            <plugin>
                <groupId>org.apache.maven.plugins</groupId>
                <artifactId>maven-compiler-plugin</artifactId>
                <version>${maven.compiler.plugin.version}</version>
                <configuration>
                    <source>${maven.compiler.source}</source>
                    <target>${maven.compiler.target}</target>
                </configuration>
            </plugin>
            <plugin>
                <groupId>org.apache.maven.plugins</groupId>
                <artifactId>maven-surefire-plugin</artifactId>
                <version>2.19.1</version>
                <configuration>
                    <suiteXmlFiles>
                        <suiteXmlFile>testng.xml</suiteXmlFile>
                    </suiteXmlFiles>
                    <printSummary>true</printSummary>
                </configuration>
            </plugin>
            <plugin>
                <groupId>com.epam.healenium</groupId>
                <artifactId>hlm-report-mvn</artifactId>
                <version>${hlm.report.mvn.version}</version>
                <executions>
                    <execution>
                        <id>hlmReport</id>
                        <phase>compile</phase>
                        <goals>
                            <goal>initReport</goal>
                        </goals>
                    </execution>
                    <execution>
                        <id>hlmReportB</id>
                        <phase>test</phase>
                        <goals>
                            <goal>buildReport</goal>
                        </goals>
                    </execution>
                </executions>
            </plugin>
        </plugins>
    </build>

    <dependencies>
        <dependency>
            <groupId>org.seleniumhq.selenium</groupId>
            <artifactId>selenium-java</artifactId>
            <version>${selenium.java.version}</version>
        </dependency>
        <dependency>
            <groupId>com.epam.healenium</groupId>
            <artifactId>healenium-web</artifactId>
            <version>${healenium.version}</version>
        </dependency>
        <dependency>
            <groupId>org.testng</groupId>
            <artifactId>testng</artifactId>
            <version>${testng.version}</version>
        </dependency>
        <dependency>
            <groupId>io.github.bonigarcia</groupId>
            <artifactId>webdrivermanager</artifactId>
            <version>${webdrivermanager.version}</version>
        </dependency>
        <dependency>
            <groupId>io.qameta.allure</groupId>
            <artifactId>allure-testng</artifactId>
            <version>${allure.version}</version>
        </dependency>
    </dependencies>

</project>
`

**Comment by Alex-Reif** [comment]
Hi @rgupta95 , @BalurQA , @priyanka748 , fixed in hlm-backend 3.3.3
